custom_components/better_thermostat/helpers.py

```diff
@@ -36,6 +38,19 @@ def convert_decimal(decimal_string):
 	except ValueError:
 		return None
 
+async def get_trv_model(self):
```

Comment:
Yeah, not liking this code split.

We don't call it anywhere else, just in models.models.get_device_model(). Just put the code into this function.

Reply:
Its now in the get_device_model function